.github/workflows/frontend-ci.yml

```diff
@@ -122,21 +134,20 @@ jobs:
     - name: Run Playwright tests
       run: |
           # start vue dev server
-          yarn --cwd ../web/ run dev 2> /dev/null &
+          ${{ matrix.mode.frontend_server_command }} &
           while ! nc -z localhost 8085; do
             sleep 3
           done
 
           # start the dandi-api server
-          python ../manage.py runserver &
+          ${{ matrix.mode.backend_server_command }} &
 
           # run the tests
-          npx playwright test
-      working-directory: e2e
+          cd e2e && npx playwright test
```

Comment:
Just curious -- why this change?

Reply:
Since we also invoke `gunicorn` in this command, setting the working-directory to `e2e` makes it more complicated to invoke. My understanding is `gunicorn --bind 0.0.0.0:8000 ../dandiapi.wsgi` wouldn't work because `../dandiapi.wsgi` isn't a valid Python module path. So we'd need to set `PYTHONPATH` explicitly to call it from `/e2e`. It seemed cleaner to just set the working-directory to the root and `cd` into `e2e/` when needed.